crypto/bls/mod.go

```diff
@@ -1,3 +1,12 @@
+// Package bls implements the cryptographic primitives using the BLS signature
+// scheme and the BN256 elliptic curve.
```

Comment:
isn't it an issue that we're still using BLS instead of BDN, considering it's broken?
It's outside the scope of this PR, but if I'm correct, you might want to add a deprecated warning here as well.

Reply:
BLS is not really broken but there is a possible attack with a rogue public key. So depending on the situation, your system can be safe in that matter.